include/app/tabs/camera.hpp

```diff
@@ -1,35 +1,60 @@
-#pragma once
-
-#include <QLabel>
-#include <QMediaPlayer>
-#include <QStackedLayout>
-#include <QString>
-#include <QVideoWidget>
-
-#include "app/config.hpp"
-#include "app/theme.hpp"
-
-class CameraTab : public QWidget {
-    Q_OBJECT
-
-   public:
-    CameraTab(QWidget *parent = nullptr);
-
-   private:
-    QWidget *connect_widget();
-    QWidget *camera_widget();
-    QWidget *input_widget();
-    void connect_stream();
-    void update_status(QMediaPlayer::MediaStatus media_status);
-
-    Theme *theme;
-    Config *config;
-    QLabel *status;
-    QMediaPlayer *player;
-    QString url;
-
-   signals:
-    void connected();
-    void disconnected();
-};
-
+#ifndef CAMERA_HPP_
+#define CAMERA_HPP_
+
+#include <QLabel>
+#include <QMediaPlayer>
+#include <QStackedLayout>
+#include <QString>
+#include <QVideoWidget>
+#include <QComboBox>
+#include <QRadioButton>
+#include <QCamera>
+#include <QCheckBox>
+#include <QCameraViewfinder>
+#include <QCameraViewfinderSettings>
+
+#include <app/config.hpp>
```

Comment:
use quotes for the path

Reply:
Standard practice is to use quotes for the current directory or a directory relative to the current directory as the only difference between the two is that quotes searches the current directory before using the include path.

Reply:
yeah i know but it was changed everywhere in this new repo so i would say just conform to how its done here

Reply:
Heh, ok